File: Java/4-PizzaStoreFactory/src/main/java/com/mycompany/pizzastorefactory/NewYorkPizzaStore/NYStyleCheesePizza.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.pizzastorefactory.NewYorkPizzaStore;

import com.mycompany.pizzastorefactory.Pizza;

/**
 *
 * @author ismas
 */
public class NYStyleCheesePizza extends Pizza {
    public NYStyleCheesePizza() {
        name = "NY Style Sauce and Cheese Pizza";
        dough = "Thin Crust Dough";
        sauce = "Marinara Sauce";
        
        toppings.add("Grated Reggiano Cheese");
    }
}

File: C#/4. PizzaStore/Classes/New York Pizza Store/NYStyleCheesePizza.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PizzaStore.Classes
{
    public class NYStyleCheesePizza: Pizza
    {
        public NYStyleCheesePizza()
        {
            _Name = "NY Style Sauce and Cheese Pizza";
            _Dough = "Thin Crust Dough";
            _Sauce = "Marinara Sauce";

            _Toppings.Add("Grated Reggiano Cheese");
        }
    }
}
